### PR TITLE
Feature/favoriting deleting ideas r

### PR DESCRIPTION
**Pull request by JRicardoT**
What (if any) features are you implementing?

-Implementing functionality to favorite cards when start-image is clicked.
-Implementing functionality to delete images when delete-image icon is clicked.
-when either images are clicked page will not reload.
What (if anything) did you refactor?
-I refactored function to render cards.
Were there any issues that arose?
-No, its a working iteration.
Is there anything that you need from your teammate?
-To review code to see if we need to refactor anything.
Any other comments, questions, or concerns?

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -4,41 +4,44 @@ var titleInput = document.querySelector('#title');
...
+  renderCard(newCard);
   clearInput();
 };
 
-function renderCard() {
-  cardContainer.innerHTML = '';
-    for (var i = 0; i < ideas.length; i++) {
+function renderCard(newCard) {
+  // console.log(newCard);
+  // cardContainer.innerHTML = '';
```
Comment (minsungpark89) [inline main.js:26]: We should get these comments deleted out once we branch out

File: main.js [hunk 2]
```diff
@@ -54,6 +57,48 @@ function checkInputFields() {
...
+    target.src = "./assets/star.svg";
+    target.alt = "white-star"
+  } else {
+    idea.isStar = true;
+    target.src = "./assets/star-active.svg"
+    target.alt = "red-star"
+  }
+};
+
+
```
Comment (tanya2108) [inline main.js:92]: Delete line

**Review by minsungpark89 — COMMENTED** [review]
I think we should strive for doing a commit after each functionality working and complete.  Probably gotta rename some variables in the near future but all in all good work!

**Review by tanya2108 — COMMENTED** [review]
Good job, refactoring and helper functions. 
Need to delete console logs and blank lines.

**Pull request merged**